Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.mm

```diff
@@ -165,6 +175,26 @@ void didReceiveAuthenticationChallenge(Ref<WebKit::AuthenticationChallengeProxy>
 
         [m_delegate.getAutoreleased() didReceiveAuthenticationChallenge:nsURLChallenge completionHandler:completionHandler.get()];
     }
+#if ENABLE(SCREEN_TIME)
+    void getScreenTimeURLs(const String& identifer, CompletionHandler<void(HashSet<URL>&&)>&& callback) const final
+    {
+        if (![PAL::getSTWebHistoryClass() instancesRespondToSelector:@selector(fetchAllHistoryWithCompletionHandler:)])
+            return;
```

Comment:
the completion handler should always be called in all cases (and should probably be named "completionHandler" instead of 'callback"